tidypolars/funs.py

```diff
@@ -131,6 +131,22 @@ def as_string(x):
     x = _col_expr(x)
     return x.cast(pl.Utf8)
 
+def as_factor(x, levels):
+    """
+    Convert to factor. Defaults to Utf8.
+
+    Parameters
+    ----------
+    x : Str
+        Column to operate on
+
+    Examples
+    --------
+    >>> df.mutate(string_x = tp.as_factor('x'))
```

Comment:
Column name should be changed to `categorical_x`

Reply:
See [https://github.com/markfairbanks/tidypolars/pull/242#discussion_r1755909332](above)

Reply:
So in this case we would use `factor_x` and `categorical_x` for the `as_categorical()` alias